radio/radio_sender.py

```diff
@@ -25,7 +25,7 @@ def encode(self, *args):
 
 
 class Sender:
-    def __init__(self, target="localhost", port=7000, source_callsign='SP3SAT', destination_callsign='PWSAT2-0'):
+    def __init__(self, target="localhost", port=7000, source_callsign='SQ5PUI', destination_callsign='PWSAT2-0'):

```

Comment:
imo source callsign should not have default value here or default value here shoud be an empty string.

Reply:
fixed